src/overmap.cpp

```diff
@@ -4328,7 +4488,7 @@ void overmap::signal_hordes( const tripoint_rel_sm &p_rel, const int sig_power )
         if( !mg.horde ) {
             continue;
         }
-        const int dist = rl_dist( absp, mg.abs_pos );
+        const int dist = 12 * rl_dist( absp, mg.abs_pos );
```

Comment:
Should this 12 be SEEX?

Reply:
Thanks for bringing this up, it was actually broken :D
Changed the 12s to SEEX and commented a bit why,